news/2017-09-10-new-featured-artist-namirin.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "New Featured Artist: *namirin"
+date: 2017-09-10 15:00:00 +0000
+---
+
+**\*namirin**, osu!'s favourite utaite, enters the fray as our newest Featured Artist with her newest mini-album, *5colors*!
+
+[![](https://assets.ppy.sh/artists/18/header.jpg)](https://osu.ppy.sh/beatmaps/artists/18)
+
+Adored by much of the _osu!_ community for her delightfully sweet voice and explosive popularity in the wake of her cover of _Yowamushi Pedal_'s Koi no Hime Hime Pettanko, **\*namirin (＊なみりん)** makes her entrance to the _osu!_ Featured Artist roster with her latest mini-album, *5colors*.
```

Comment:
`in the wake of her cover of` there is a short repetition of the word "of"... just pointing this out, but I have nothing better to suggest